@ecomplus/storefront-components/src/js/AccountForm.js

```diff
@@ -0,0 +1,224 @@
+import {
+  _config,
```

Comment:
```suggestion
  $ecomConfig,
```

Não usamos mais o `_config` do utils (é uma variável interna, por isto o underline inclusive), note que nos outros componentes também sempre tem o `$ecomConfig`...